tests/ui/invalid_intopy_with.stderr

```diff
@@ -7,8 +7,8 @@ error[E0308]: mismatched types
 5 |     #[pyo3(into_py_with = into)]
```

Comment:
You're probably using an older compiler than CI. In any case, I think you can drop this changed file from the PR and it should just all work.